src/css/components/panel.scss

```diff
@@ -121,3 +121,17 @@ h4 + .panel {
   flex-shrink: 1;
   min-width: 25%;
 }
+
+.panel-new {
```

Comment:
When I have the chance, I'm going to write a TODO to remove this when we finalize the panels, because I don't like having separate code paths for new and old things.